### PR TITLE
docs: updated index.md

### PR DESCRIPTION
**Pull request by k-avy**
## Project

License as a Service

### Changes

Added the basic information about the project in index.md.

### CODE REVIEW

File: docs/2023/laas/index.md [hunk 1]
```diff
@@ -15,9 +15,31 @@ SPDX-FileCopyrightText: 2023 Kavya Shukla <kavyuushukla59@gmail.com>
...
+can be challenging to comprehend in terms of how they relate to a particular piece
+of software or work with other licences. Organisations like FOSSology and SW360
+can utilise it for a variety of things, like licence identification, filtering,
+and management.
+
 ## What should be done?
+
+There are two parts of the project:
+
+- First part is the creating a datebase service
```
Comment (GMishx) [inline docs/2023/laas/index.md:35]:
```suggestion
- First part is the creating a database service
```

**Review by GMishx — CHANGES_REQUESTED** [review]

File: docs/2023/laas/index.md [hunk 2]
```diff
@@ -15,9 +15,31 @@ SPDX-FileCopyrightText: 2023 Kavya Shukla <kavyuushukla59@gmail.com>
...
+  - Adding the data components related to licenses.
+  - Database will help us to fetch the desired data using the API.
+
+- Second part is API Designing:
+  - Adding endpoints related to the license database.
+  - Adding various query parameters as a filter.
```
Comment (avinal) [inline docs/2023/laas/index.md:41]:
```suggestion
  - Adding various query parameters for filters and ordering.
```

File: docs/2023/laas/index.md [hunk 3]
```diff
@@ -15,9 +15,31 @@ SPDX-FileCopyrightText: 2023 Kavya Shukla <kavyuushukla59@gmail.com>
...
+  - To add fuzzy search the license text.
```
Comment (avinal) [inline docs/2023/laas/index.md:42]:
```suggestion
  - To add fuzzy search for the license text.
```

**Review by avinal — CHANGES_REQUESTED** [review]
Minor changes

**Pull request merged**